### DISCUSSION TITLE
How to disable [info] from module in error.log

### DISCUSSION THREAD

**Discussion by severt** [stnoonan/spnego-http-auth-nginx-module#126] [Q&A] [answered]
I receive multiple messages from the module
what nginx settings to disable them?
2022/03/02 13:42:50 [info] 2588664#0: auth_spnego: fqun = 0 in /etc/nginx/nginx.conf:98
2022/03/02 13:42:50 [info] 2588664#0: auth_spnego: allow_basic = 1 in /etc/nginx/nginx.conf:98
2022/03/02 13:42:50 [info] 2588664#0: auth_spnego: force_realm = 0 in /etc/nginx/nginx.conf:98
2022/03/02 13:42:50 [info] 2588664#0: auth_spnego: map_to_local = 0 in /etc/nginx/nginx.conf:98
2022/03/02 13:42:50 [info] 2588664#0: auth_spnego: delegate_credentials = 0 in /etc/nginx/nginx.conf:98
2022/03/02 13:42:50 [info] 2588664#0: auth_spnego: constrained_delegation = 0 in /etc/nginx/nginx.conf:98

**Comment by stnoonan** [comment]
It looks like you probably have NGX_DEBUG true in your build?
https://github.com/stnoonan/spnego-http-auth-nginx-module/blob/c62616386e31c65eff648bb5524f7928286a5a0e/ngx_http_auth_spnego_module.c#L340

**Comment by Alphix** [comment, marked as answer]
This was fixed in https://github.com/stnoonan/spnego-http-auth-nginx-module/commit/ea8995e498be153a2a9a574ba5450b3a27dcc3ef
